spring-data-microservice-transaction-sample/rpc/src/main/proto/sample.proto

```diff
@@ -73,6 +73,9 @@ service CustomerService {
   // Get a customer information
   rpc GetCustomerInfo(GetCustomerInfoRequest) returns (GetCustomerInfoResponse) {
   }
+  // Get a customer information
```

Comment:
```suggestion
  // Get customer information
```

Reply:
@josh-wong Thanks for the suggestions! I'm merging them.